inst/templates/study_schema.R

```diff
@@ -0,0 +1,28 @@
+#' Study Schema for {{ study_name }}
+#'
+#' To source the study schema in a chunk in your Rmarkdown report:
+#' source("R/study_schema.R")
+#'
+#' @return a kable table with the study schema
+#' @export
+#'
+#' @examples
+study_schema <- function() {
+
+  caption <- "{{ study_name }} study schema."
+
+  schema_table <- tibble::tribble(
+    ~Group, ~`Sample Size`, ~`Week 10`, ~`Week 20`,
+    "Group A", 10, "Dose A", "Dose A",
+    "Group B", 10, "Dose B", "Dose B"
+    )
+
+  schema_table %>%
+    knitr::kable(
+      format = output_type,
+      caption = caption,
+      booktabs = TRUE,
+      linesep = ""
```

Comment:
Making these two params with defaults TRUE and "", respectively, would make this function more flexible.

Reply:
Would this require changes? I'm not sure I understand your comment.

Reply:
Would this require changes? I'm not sure I understand your comment.

Reply:
Ah, OK, reading this and I think I understand better. The reason why we'd make these params is if the study schema needs to change between PT reports. I think in those cases, the user could edit this function to make it more flexible if they need it to be. Does that make sense?

Reply:
also I can see this function directly, where you want a nice default study schema kable code

Reply:
but you are right this change isn't vital since it's easy to change the code after